test/test_secret_manager_with_cli.py

```diff
@@ -0,0 +1,52 @@
+import requests
+import json
+import argparse
+import base64
+import time
+import random
+import hmac
+import os
+from hashlib import sha256
+from collections import OrderedDict
+from cli import createkey, enroll, createSecret
+import urllib.parse
+import _utils_
+appid= ''
+apikey= ''
+keyid= ''
+
+def get_appid_apikey(base_url):
+    global appid
+    global apikey
+    appid, apikey = enroll.enroll(base_url)
+    _utils_.init_appid_apikey(appid, apikey)
+
+def test_secret_manager(base_url, headers):
+    
+    print('====================test_secret_manager start===========================')
+    key1 = createkey.createkey(base_url, "EH_AES_GCM_128", "EH_INTERNAL_KEY")
+
+    createSecret.createSecret(base_url, "secretData1", "secret001", key1, "mysecret", "30h")
+
```

Comment:
you should add another case without provide the cmk.

Reply:
it will be add in test